spec/imports/imports.spec.js

```diff
@@ -16,97 +16,52 @@ describe('imports section', function() {
-            browser.sleep(200).then(done);
-        });
-        it('should orderBy type', function(done) {
-            components.imports.page.orderByNodeType();
-            expect(components.imports.page.checkFirstNodeType()).toBe('cloudify.puppet.nodes.WebServer');//last node type
-
-            browser.sleep(200).then(done);
-        });
-        it('should change used field for node types', function(done) {
-            components.layout.goToTopology();
-            //add node on topology page
-            var dragItem = components.topology.page.findSpecificNodeContainerOnTopologyPage('cloudify.docker', 'Container');
-            var dragDest = element(by.id('topologyContainer'));
-            components.layout.dragAndDrop(dragItem, dragDest);//add node
-
-            components.layout.goToImports();
-            expect(components.imports.page.findSpecificUsedOnImportsPage().getText()).toBe('1');//node types 'docker' used field
-
-            components.layout.goToTopology();
-            //delete node on topology page
-            element(by.css('.nodeContainer')).click();
-            element(by.css('#deleteBtn')).click();
-            components.popovers.popover.clickYes();
-
-            components.layout.goToImports();
-            expect(components.imports.page.findSpecificUsedOnImportsPage().getText()).toBe('0');//node types 'docker' used field
-
-            browser.sleep(200).then(done);
-        });
-        it('should remove node types if import was deleted', function(done) {
-            components.imports.page.openSpecificDeletePopover(components.imports.page.findSpecificUrl()); //click delete icon
-            components.popovers.popover.clickYes();//click no btn
-            expect(components.layout.isElementPresent(components.imports.page.findSpecificUsedOnImportsPage())).toBe(false);
-
-
-            browser.sleep(200).then(done);
-        });
-    });
     describe('Save imports', function() {
         it('should add and save imports', function(done){
+            components.layout.goToTopology();
             components.imports.page.openAddImportsDialog();
-            components.modals.modal.enterUrl('http://getcloudify.org.s3.amazonaws.com/spec/chef-plugin/1.3.1/plugin.yaml', 'imports');
+            components.modals.modal.enterUrl('http://getcloudify.org.s3.amazonaws.com/spec/docker-plugin/1.3/plugin.yaml', 'imports');
             components.modals.modal.save();
 
             components.layout.saveBlueprint(); //click save blueprint button
-            browser.refresh();
-
-            expect(components.layout.countElements(element.all(by.repeater('imports in globals.stencils.importsData')))).toBe(10);
-
             browser.sleep(200).then(done);
 
+            browser.driver.navigate().refresh();
+            expect(components.layout.countElements(element.all(by.repeater('imports in globals.stencils.importsData')))).toBe(2);
```

Comment:
Why do we need to refresh? doesnt it happen update the table? why isnt wait good enough?